generators/server/templates/src/main/java/package/config/Constants.java.ejs

```diff
@@ -32,6 +32,7 @@ public final class Constants {
     <% if (databaseType === 'couchbase') { %>
     public static final String ID_DELIMITER = "__";
     <% } %>
+    public static final String CLOUD_CONFIGURATION_PREFIX = "spring.datasource.hikari";
```

Comment:
This is to apply the properties specified in the configuration file. When using connectors in the classpath, while it creates the DS, it won't consider boot properties. And JBP (build pack) also won't create the DS when we do a cf push as it would have already been taken care by connectors. This prefix is necessary to force the connectors to honour the config properties.

Reply:
Oh I see, this is to have a constant here, probably to make Sonar happy... This is quite useless, I'm going to refactor this

Reply:
Thanks @srinivasa-vasu - I was refactoring this while you posted the comment, can you have a look at my changes? I will squash the commits when I merge

Reply:
@jdubois looks good. Thnx.

Reply:
Awesome, let's do this :-)
Thanks a lot @srinivasa-vasu !